Update index.md (#29278)
dotnet/docs: docs/core/porting/index.md

## The future of .NET Standard

> .NET Standard is a formal specification of .NET APIs that are available on multiple .NET implementations. The motivation behind .NET Standard was to establish greater uniformity in the .NET ecosystem. Starting with .NET 5, a different approach to establishing uniformity has been adopted, and this new approach eliminates the need for .NET Standard in many scenarios. For more information, see [.NET and .NET Standard](../../standard/net-standard.md#net-5-and-net-standard).

Reviewer: ```suggestion
.NET Standard is a formal specification of .NET APIs that are available on multiple .NET implementations. The motivation behind .NET Standard was to establish greater uniformity in the .NET ecosystem. Starting with .NET 5, a different approach to establishing uniformity has been adopted, and this new approach eliminates the need for .NET Standard in many scenarios. For more information, see [.NET 5+ and .NET Standard](../../standard/net-standard.md#net-5-and-net-standard).
```

✔️ CONSIDER using the [.NET Upgrade Assistant](upgrade-assistant-overview.md) to migrate your projects. Even though this tool is in preview, it automates most of the manual steps detailed in this article and gives you a great starting point for continuing your migration path.

> ✔️ CONSIDER examining your dependencies first. Your dependencies must target .NET 6, .NET Standard, or .NET Core.

Reviewer: This is about porting apps to .NET, if it's OK for dependencies to target .NET Core it must be OK to target .NET 5, which is later than Core.

```suggestion
✔️ CONSIDER examining your dependencies first. Your dependencies must target .NET 5+, .NET Standard, or .NET Core.
```

✔️ DO migrate from a NuGet _packages.config_ file to [PackageReference](/nuget/consume-packages/package-references-in-project-files) settings in the project file. Use Visual Studio to [convert the _package.config_ file](/nuget/consume-packages/migrate-packages-config-to-package-reference#migration-steps).

✔️ CONSIDER upgrading to the latest project file format even if you can't yet port your app. .NET Framework projects use an outdated project format. Even though the latest project format, known as SDK-style projects, was created for .NET Core and beyond, they work with .NET Framework. Having your project file in the latest format gives you a good basis for porting your app in the future.

✔️ DO retarget your .NET Framework project to at least .NET Framework 4.7.2. This ensures the availability of the latest API alternatives for cases where .NET Standard doesn't support existing APIs.

> ✔️ CONSIDER targeting .NET 6 is LTS.

Reviewer: ```suggestion
✔️ CONSIDER targeting .NET 6, as it's a long-term support (LTS) release.
```

> ✔️ DO target .NET 6 for **Windows Forms and WPF** projects. .NET 6 contains many improvements for Desktop apps.

Reviewer: ```suggestion
✔️ DO target .NET 5+ for **Windows Forms and WPF** projects. .NET 5 and later releases contain many improvements for Desktop apps.
```

Author: this is fine for now but as support ends for .Net in three days maybe just say go with .Net 6

Author: or 6+
